### ISSUE TITLE
Feature request: launch with command line parameters

### ISSUE THREAD

**Issue by maxwinkler13** [subhra74/snowflake#59]
Awesome software, lots of potential. What would make this a lot more useful though would be if it could be launched directly via command line parameters, like putty for example.
That would enable the tool to be included within other remote management suites, like RDM or RoyalTS for example.
It should also be able to read from other sources of configured connections to reduce dual maintenance between solutions. Currently you'd have to create sessions for snowflake explicitly, as far as I can see. If I'm incorrect, where would I find more info about this?

**Comment by subhra74** [comment]
You are right, import export is not there yet, but will be coming in future releases. Adding command line suggestion to backlog
